<?xml version="1.0"?>
<launch>

  <node pkg="path_smoothing" type="traj_smoothing_service.py" name="traj_smoothing_service" />
  <node pkg="path_smoothing" type="path_smoothing_service.py" name="path_smoothing_service" />
</launch>
